### PR TITLE
Hello-Rails-Back-end

### PR DESCRIPTION
**Pull request by Terbeche**
### I've followed the following requirements
- Create a new Rails API app called 'hello-rails-back-end'.
- Initialize your project with Git.
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,77 @@
...
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> This is the backend side of a Hello-World web-app using Rails.
 
-Things you may want to cover:
+# Hello-React-Front-End
+[React Back-end app](https://github.com/Terbeche/hello-react-front-end/tree/dev)
 
-* Ruby version
+## Built With
```
Comment (BenMukebo) [inline README.md:8]: - [x] Kindly update your README file actually you're missing a list of technologies used in this project, let's add it.

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Terbeche,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Professional README ✔️ 
- You have followed  JavaScript best practices. ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — CHANGES_REQUESTED** [review]
...

File: README.md (hunk 1 above)
Comment (rloterh) [reply README.md:8]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Mustapha,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾  Congratulations!  🎉💯🌟 

## Highlights 👍
- React frontend connects correctly to Rails API backend
- Good ReadMe

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
- You used incorrect gitflow/ GitHub flow for this project. Kindly note that the merge request for this project should flow from **from feature into development**. Kindly note this important step for subsequent projects.

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**